plugins/outputs/influxdb/http.go

```diff
@@ -592,3 +597,18 @@ func makeQueryURL(loc *url.URL) (string, error) {
 func (c *httpClient) Close() {
 	c.client.CloseIdleConnections()
 }
+
+type wrappedReader struct {
+	r            io.ReadCloser
+	bytesWritten atomic.Int64
+}
+
+func (r *wrappedReader) Read(p []byte) (int, error) {
+	n, err := r.r.Read(p)
+	r.bytesWritten.Add(int64(n))
```

Comment:
The `wrappedReader` tracks bytes during read operations, but the stat is only updated after the write completes. Since reads may happen concurrently, the atomic counter ensures correctness.
Can we add a comment here for clarity?
```suggestion
    // Atomically update bytesWritten to ensure thread-safe tracking during concurrent reads
    r.bytesWritten.Add(int64(n))
```